server/lib/base_konnector.js

```diff
@@ -2,6 +2,7 @@ const _ = require('lodash')
 const printit = require('printit')
 const slugify = require('cozy-slug')
 const fetcher = require('./fetcher')
+const package = require('../../package.json')
```

Comment:
Does not compile: 

SyntaxError: server/lib/base_konnector.js: package is a reserved word in strict mode (5:6)
  3 | const slugify = require('cozy-slug')
  4 | const fetcher = require('./fetcher')
> 5 | const package = require('../../package.json')